channels/uk.m3u

```diff
@@ -379,8 +379,6 @@ https://vod365-planet-knowledge-1-gb.samsung.wurl.com/manifest/playlist.m3u8
 https://pocketwatch-gb.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="QelloConcertsbyStingray.uk" tvg-name="Qello Concerts by Stingray" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/IVUgJPY.png" group-title="Music",Qello Concerts by Stingray (1080p)
 https://stingray-qelloconcerts-1-gb.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="QelloConcertsbyStingray.uk" tvg-name="Qello Concerts by Stingray" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/IVUgJPY.png" group-title="Music",Qello Concerts by Stingray (1080p) [Offline]
-https://csm-e-stv.tls1.yospace.com/csm/live/211935407.m3u8
```

Comment:
Well... it's an odd one.
Works, but redirects to https://stv.ott-channels.stingray.com/delta-qello/master.m3u8